projects/angular/src/utils/i18n/common-strings.default.ts

```diff
@@ -84,8 +84,8 @@ export const commonStringsDefault: ClrCommonStrings = {
   comboboxNoResults: 'No results',
   comboboxOpen: 'Show options',
   // Datagrid expandable rows
-  dategridExpandableBeginningOf: 'Beginning of',
-  dategridExpandableEndOf: 'End of',
-  dategridExpandableRowContent: 'Expandable row content',
-  dategridExpandableRowsHelperText: `Screen reader table commands may not work for viewing expanded content, please use your screen reader's browse mode to read the content exposed by this button`,
+  datagridExpandableBeginningOf: 'Beginning of',
+  datagridExpandableEndOf: 'End of',
+  datagridExpandableRowContent: 'Expandable row content',
+  datagridExpandableRowsHelperText: `Screen reader table commands may not work for viewing expanded content, please use your screen reader's browse mode to read the content exposed by this button`,
```

Comment:
There's no need to add the `dategrid*` stuff here. Because all that work above was just to prevent a breaking change to the folks who were using it before...